docs/contributing.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+OpenSAFELY is an open source project; all our code is available for inspection and reuse, and we can accept contributions in the following ways:
+
+## Documentation
+
+Typos and other small improvements are always very welcome. Raise an issue, or even better a Pull Request, in the [documentation repo](https://github.com/opensafely/documentation/). We will incorporate these contributions after we've checked them for consistency with our house style.
+
+We welcome proposals for longer topic-based guides. Suggest them as an [issue in the documentation repo](https://github.com/opensafely/documentation/issues), and we'll work with you to define an overall structure and format, before you write and submit the draft.
+
+## Reusable actions
+These are units of software that solve a problem for several studies without the need to copy-and-paste between them, [described in more detail here](./actions-reusable.md).  They can be shared between researchers, even between groups that use different programming languages, and are one of the best ways you can make contributions that benefit the community.  Because they have a clear, pluggable interface with the rest of OpenSAFELY, it's easy for us to accept new reusable actions into our [actions library](https://actions.opensafely.org/). If you've written a reusable action you'd like to contribute to the actions library, please get in touch at [team@opensafely.org](mailto:team@opensafely.org).
```

Comment:
```suggestion
These are units of software that solve a problem for several studies without the need to copy-and-paste between them, [described in more detail here](./actions-reusable.md).  They can be shared between researchers, even between groups that use different programming languages, and are one of the best ways you can make contributions that benefit the community. If you've written a reusable action you'd like to contribute to the actions library, please get in touch at [team@opensafely.org](mailto:team@opensafely.org).
```

I don't think we should accept anything as a reusable action, so would emphasize getting in touch over how "easy" it is for us to accept new reusable actions.

Reply:
I'll add an issue to the data team backlog for documenting reusable action "best practice".